nucleus/common/internal-api/src/main/java/org/glassfish/internal/data/ModuleInfo.java

```diff
@@ -287,7 +287,7 @@ public synchronized void start(
             }
             started=true;
             if (events!=null) {
-                DeploymentSpan innerSpan = tracing.startSpan(DeploymentTracing.AppStage.START_EVENTS, "Module");
+                DeploymentSpan innerSpan = tracing.startSpan(DeploymentTracing.AppStage.PROCESS_EVENTS, Deployment.MODULE_STARTED.type());
                 events.send(new Event<ModuleInfo>(Deployment.MODULE_STARTED, this), false);
```

Comment:
Shouldn't MODULE_LOADED event also be traced like MODULE_STARTED, APPLICATION_LOADED, and APPLICATION_STARTED.